msticpy/sectools/vtlookupv3.py

```diff
@@ -229,59 +261,75 @@ def lookup_ioc_relationships(self,
         KeyError
             Unknown vt_type
         """
-        if VTEntityType(vt_type) not in self.supported_vt_types:
+        if VTEntityType(vt_type) not in self._SUPPORTED_VT_TYPES:
             raise KeyError(f"Property type {vt_type} not supported")
 
         endpoint_name = self._get_endpoint_name(vt_type)
+        response: vt.object.Object
 
         if limit is None:
-            endpoint_name = self._get_endpoint_name(VTEntityType(vt_type))
```

Comment:
I deleted 'endpoint_name = self._get_endpoint_name(VTEntityType(vt_type))' here since it's already populated on line 267 (prev 235)